### PR TITLE
Have option to specify gitops path where to place generated files

### PR DESCRIPTION
**Pull request by arturo-skydio**
## Description

Currently, gitops files get generated under a hardcoded prefix path of cloud, need a way to have a variable input for this

## Related Issue

https://github.com/adobe/rules_gitops/issues/60

## Motivation and Context

For repositories that do not want gitops files generated at `cloud` path, this change is needed

## How Has This Been Tested?

Just started using this and verified that no gitops_path renders gitops files in usual `cloud` directory, while specifying path correctly changes the prefix path

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by michaelschiff** [comment]
@arturo-skydio thank you for the contribution! mind signing the CLA?

**Comment by arturo-skydio** [comment]
> @arturo-skydio thank you for the contribution! mind signing the CLA?

@michaelschiff I thought I did,
not sure if its because the change comes from our skydio org, maybe I should just try to open the PR from my gh account?

**Comment by arturo-skydio** [comment]
closed in favor of #62

**Comment by arturo-skydio** [comment]
@michaelschiff sorry, was a bit impatient with opening up the other PR, looks like the CLA is green now, thanks

**Review by kzadorozhny — CHANGES_REQUESTED** [review]
Looks good overall. No issue with the change. But there are couple issues that need to be addressed before we can merge:
- You might want to add another test case into the [e2e_test](https://github.com/adobe/rules_gitops/blob/master/examples/e2e-test.sh). Right now we rely on `cloud` directory being default. For example you can add another `k8s_deploy` target alongside `release` and `canary` targets and update the script. Don't forget to update the `.gitignore` file too.
- `cloud` also is hardcoded in couple places ( [1](https://github.com/adobe/rules_gitops/blob/master/gitops/git/git.go#L97), [2](https://github.com/adobe/rules_gitops/blob/master/gitops/git/git.go#L45) ) in the [GitOps PRs tool](https://github.com/adobe/rules_gitops/tree/master/gitops/prer).

**Comment by arturo-skydio** [comment]
> Looks good overall. No issue with the change. But there are couple issues that need to be addressed before we can merge:
> - You might want to add another test case into the [e2e_test](https://github.com/adobe/rules_gitops/blob/master/examples/e2e-test.sh). Right now we rely on `cloud` directory being default. For example you can add another `k8s_deploy` target alongside `release` and `canary` targets and update the script. Don't forget to update the `.gitignore` file too.
> - `cloud` also is hardcoded in couple places ( [1](https://github.com/adobe/rules_gitops/blob/master/gitops/git/git.go#L97), [2](https://github.com/adobe/rules_gitops/blob/master/gitops/git/git.go#L45) ) in the [GitOps PRs tool](https://github.com/adobe/rules_gitops/tree/master/gitops/prer).
> 

Sounds good, thanks for taking a look, I'll add the test

**Comment by arturo-skydio** [comment]
@kzadorozhny Added a test for `gitops_path` and made sure to remove hardcoding in the Gitops Prs tool, let me know if all look good, thanks again

**Review by kzadorozhny — CHANGES_REQUESTED** [review]
Overall LGTM. Please address one comment and I'll merge it.

File: examples/helloworld/BUILD [hunk 1]
```diff
@@ -111,6 +131,7 @@ sh_test(
     ],
     data = [
         ":canary.show",
+        ":gitops_custom_path",
```
Comment (kzadorozhny) [inline examples/helloworld/BUILD:134]: Please, remove this line. You are not assertign anything from the `gitops_custom_path` output in `k8s_deploy_test.sh`.

Comment (arturo-skydio) [reply examples/helloworld/BUILD]: cool, done, wasn't sure what this was for

**Review by kzadorozhny — APPROVED** [review]

**Pull request merged**